ios-cargo

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
```

Comment:
This file should have a `.py` file extension. Maybe it should also be called `build-ios`? But the current name is fine too.

Reply:
I decided not to use the .py extension so it can be put into the bin folder & used as command

Reply:
it only uses builtin libs